src/App.tsx

```diff
@@ -31,25 +31,18 @@ import SpinBox from "./components/SpinBox";
 import HoverTooltip from "./components/HoverTooltip";
 import Export from "./components/Export";
 import DrawModeDropdown from "./components/DrawModeDropdown";
+import CanvasWrapper from "./components/CanvasWrapper";
+import LabeledRangeSlider from "./components/LabeledRangeSlider";
+import PlotWrapper from "./components/PlotWrapper";
 
 function App(): ReactElement {
   // STATE INITIALIZATION /////////////////////////////////////////////////////////
   const theme = useContext(AppThemeContext);
 
-  const canvasRef = useRef<HTMLDivElement>(null);
-  const plotRef = useRef<HTMLDivElement>(null);
-
-  const [plot, setPlot] = useState<Plotting | null>(null);
   const canv = useConstructor(() => {
     return new ColorizeCanvas();
   });
```

Comment:
Mentioned this in the description, but canv is still kept top-level here since some of its functionality is important for timing. I guess I *could* pass it up via a ref from `CanvasWrapper.tsx` but I think this might be cleaner?

Reply:
This is cleaner.